pkg/storage/storage_test.go

```diff
@@ -8,6 +8,9 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+func TestInterface(t *testing.T) {
+	assert.Implements(t, (*Storage)(nil), &LocalStorage{})
```

Comment:
This should be tested here:
https://github.com/sourcepods/sourcepods/blob/31766d2c90d98e962b5ac171c9912f2cb65296e0/cmd/storage/storage.go#L172

At this point the LocalStorage needs to implement that interface in code. Not entirely sure when there are multiple implementations, or if it fails at runtime only?!

We'll keep this anyway. More a comment on this test in general.

Reply:
I usually only test the package I'm in, only doing `make test` before commiting. Hence this test.

But true, that line will essentially test the assertion on `go build` :)